Platform.PCL/Realm.PCL/RealmPCL.cs

```diff
@@ -293,13 +293,15 @@ public void Write(Action action)
         }
 
         /// <summary>
-        /// Execute an action inside a temporary <see cref="Transaction"/> on a worker thread. If no exception is thrown,
+        /// Execute an action inside a temporary <see cref="Transaction"/> on a worker thread, <b>if</b> called from UI thread. If no exception is thrown,
```

Comment:
I don't think I did - it was merged in this branch which is why you're seeing it compared to master :P